src/features/search/helpers/useSortedSearchCategories/useSortedSearchCategories.tsx

```diff
@@ -21,6 +21,7 @@ export const useSortedSearchCategories = (
       },
       baseColor: category.baseColor,
       gradients: category.gradients,
+      position: category.position,
     }))
-    .sort((a, b) => a.label.localeCompare(b.label))
+    .sort((a, b) => a.position - b.position)
```

Comment:
Puisque `position` peut être `undefined`, surtout si l'API ajoute des catégories.

```suggestion
    .sort((a, b) => {
      if (a.position === undefined) return 0
      if (b.position === undefined) return 0
      return a.position - b.position
    })
```

Je ne sais pas si ça ferait le taf précisément mais il faut jouer avec pour trouver quelque chose qui marche.

Reply:
Après peut-être que ce que tu avais fait fonctionnait. Mais je veux bien voir le test du genre

```
const items = {
  toto: { position: 1 },
  tata: { position: undefined },
  titi: { position: 2 }
}
```

et vérifier qu'on a bien `["toto", "titi", "tata"]`